packages/SwingSet/src/kernel/virtualObjectManager.js

```diff
@@ -106,17 +106,16 @@ export function makeCache(size, fetch, store) {
 /**
  * Create a new virtual object manager.  There is one of these for each vat.
  *
- * @param {*} syscall  Vat's syscall object, used to access the `vatstoreGet` and
- *    `vatstoreSet` operations.
+ * @param {*} syscall  Vat's syscall object, used to access the `vatstoreGet`
+ *   and `vatstoreSet` operations.
  * @param {() => number} allocateExportID  Function to allocate the next object
- *    export ID for the enclosing vat.
- * @param valToSlotTable {*}  The vat's table that maps object identities to their
- *    corresponding export IDs
- * @param m {*}  The vat's marshaler.
- * @param cacheSize {number}  How many virtual objects this manager should cache
+ *   export ID for the enclosing vat.
+ * @param {*} valToSlotTable  The vat's table that maps object identities to
+ *   their corresponding export IDs
+ * @param {*} m The vat's marshaler.
+ * @param {number} cacheSize How many virtual objects this manager should cache
  *   in memory.
- *
- * @returns a new virtual object manager.
+ * @returns {*} a new virtual object manager.
```

Comment:
@FUDCo I adjusted this to appease the linter. It appears to have become more picky.